utils/thrift/thrift/protocol/TCompactProtocol.tcc

```diff
@@ -808,8 +808,6 @@ TType TCompactProtocolT<Transport_>::getTType(int8_t type) {
     case detail::compact::CT_STRUCT:
       return T_STRUCT;
     default:
-
-      cout << __FILE__<< __LINE__ << __func__<< endl;
```

Comment:
Are you sure this relates to MCOL-641?

Reply:
Yes, this got added by mistake in the big MCOL-641 PR: https://github.com/mariadb-corporation/mariadb-columnstore-engine/pull/1604